samples/amber.cc

```diff
@@ -151,8 +151,11 @@ bool ParseArgs(const std::vector<std::string>& args, Options* opts) {
       opts->pipeline_create_only = true;
     } else if (arg == "-d") {
       opts->disable_validation_layer = true;
+      // -s is an old option which is still recognized but does nothing.
```

Comment:
Well, it does something: It reverses the -q option...

Reply:
done